controllers/storagecluster/storagecluster_controller.go

```diff
@@ -251,7 +262,16 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		).
 		Watches(&storagev1.StorageClass{}, enqueueStorageClusterRequest).
 		Watches(&volumesnapshotv1.VolumeSnapshotClass{}, enqueueStorageClusterRequest).
-		Watches(&ocsv1alpha1.StorageConsumer{}, enqueueStorageClusterRequest, builder.WithPredicates(storageConsumerStatusPredicate)).
+		Watches(
```

Comment:
We don't want to trigger for every consumer, in the reconcile you are placing owner and change this to own.

Reply:
Every consumer is owned by the storageCluster we would still trigger reconcile for every cosumer